src/main/java/com/blamejared/crafttweaker/impl_native/world/ExpandBiome.java

```diff
@@ -0,0 +1,141 @@
+package com.blamejared.crafttweaker.impl_native.world;
+
+import com.blamejared.crafttweaker.api.annotations.ZenRegister;
+import com.blamejared.crafttweaker_annotations.annotations.Document;
+import com.blamejared.crafttweaker_annotations.annotations.NativeTypeRegistration;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraft.world.biome.Biome;
+import org.openzen.zencode.java.ZenCodeType;
+
+@ZenRegister
+@Document("vanilla/api/world/MCBiome")
+@NativeTypeRegistration(value = Biome.class, zenCodeName = "crafttweaker.api.world.MCBiome")
+public class ExpandBiome {
+    
+    
+    @ZenCodeType.Getter("waterColor")
+    public static int getWaterFloat(Biome internal) {
+        return internal.getWaterColor();
+    }
+    
+    @ZenCodeType.Getter("waterFogColor")
+    public static int getWaterFogColor(Biome internal) {
+        return internal.getWaterFogColor();
+    }
+    
+    //    @ZenCodeType.Getter("defaultTemperature")
+    //    public static float getDefaultTemperature(Biome internal) {
+    //        return internal.getDefaultTemperature();
+    //    }
+    
+    //    @ZenCodeType.Getter("translationKey")
+    //    public static String getTranslationKey(Biome internal) {
+    //        return internal.getTranslationKey();
+    //    }
+    //
```

Comment:
Why are these commented out?

Reply:
Because they were also commented out in the original MCBiome file.
I moved the original class and changed the signature, so I left these in there as they were.